package.cmake

```diff
@@ -33,7 +33,8 @@ execute_process(
     ERROR_QUIET
 )
 
-list(APPEND CPACK_SOURCE_IGNORE_FILES ".git")
+list(APPEND CPACK_SOURCE_IGNORE_FILES "/.git")
```

Comment:
This should still be .git I believe. Otherwise files like .gitignore will be included